unit_tests/stencil_composition/sid/test_sid_concept.cpp

```diff
@@ -105,7 +105,7 @@ namespace gridtools {
                 std::is_same<std::decay_t<decltype(sid::get_origin(std::declval<testee &>())())>, testee *>(), "");
             static_assert(std::is_same<decltype(sid::get_strides(testee{})), strides>(), "");
 
-            GT_CONSTEXPR auto stride = sid::get_stride<void>(strides{});
+            auto stride = sid::get_stride<void>(strides{});
             static_assert(stride == 0, "");
```

Comment:
how does this static_assert work, if it is not constexpr?

Reply:
it works, probably by accident. changed anyway.